src/ucp/core/ucp_request.c

```diff
@@ -45,6 +45,19 @@ ucs_status_t ucp_tag_recv_request_test(void *request, ucp_tag_recv_info_t *info)
     return status;
 }
 
+ucs_status_t ucp_stream_recv_request_test(void *request, size_t *length)
+{
+    ucp_request_t *req   = (ucp_request_t*)request - 1;
+    ucs_status_t  status = ucp_request_check_status(request);
+
+    if (status != UCS_INPROGRESS) {
```

Comment:
this does not match the documentation " The value is valid only if the status is UCS_OK."
should be if status==UCS_OK

Reply:
Disagree, if status indicates an error, value can be invalid. We have the same logic in `tag`

Reply:
ok